cypress/integration/test.js

```diff
@@ -29,6 +28,24 @@ describe('MovieList', () => {
   it('Should show 40 movie posters', () => {
     cy.get('.poster').should('have.length', 40);
   });
+
+  it('Should have an image for each poster', () => {
+    cy.get('.poster').each(poster => {
+      cy.get(poster).get('img').should('have.class', 'poster-img')
+    });
+  });
+
+  it('Should have a rating for each poster', () => {
+    cy.get('.poster').each(poster => {
+      cy.get(poster).get('p').should('contain', 'Freshness:')
+    });
+  });
+
+  it('Should have an image for each poster', () => {
+    cy.get('.poster').each(poster => {
+      cy.get(poster).get('h2').should('be.visible')
+    });
+  });
```

Comment:
Great job with the unit testing here to make sure that a User can see everything on the page.